broadinstitute/single_cell_portal_core
# updating study sidebar and settings UX (SCP-4130)

SCP-4130 This incorporates the suggestions of Joy and the rest of the team from demo into the Study Description, settings, and study details pages.  

Study description
![image](https://user-images.githubusercontent.com/2800795/156693065-e7e68cca-4eed-4c53-9f01-a96400514ad9.png)

Study settings
![image](https://user-images.githubusercontent.com/2800795/156693674-04cedd46-e249-486d-bb1c-2bff0f3a4cd9.png)

Study details
![image](https://user-images.githubusercontent.com/2800795/156693325-4a9f21f3-db19-466c-9339-e08ee54617b5.png)

TO TEST:
1. Load a study that has some of authors, publications, or external resources
2. Confirm the sidebar appears on the study summary tab
3. Confirm the sidebar can open and close
4. Go to study settings page, and observe the "Management pages" links appear
5. Go to study details, and confirm the creation date and creator appear correctly.
